src/styles/themes/default.ts

```diff
@@ -81,16 +82,14 @@ const darkTheme = {
     skeletonLHNIn: colors.darkBorders,
     skeletonLHNOut: colors.darkDefaultButton,
     QRLogo: colors.green400,
-    starDefaultBG: 'rgb(254, 228, 94)',
-};
+} satisfies ThemeColorsWithoutPageBackgroundColors;
 
-darkTheme.PAGE_BACKGROUND_COLORS = {
-    [SCREENS.HOME]: darkTheme.sidebar,
-    [SCREENS.SETTINGS.PREFERENCES]: colors.blue500,
-    [SCREENS.SETTINGS.WORKSPACES]: colors.pink800,
-    [ROUTES.SETTINGS_STATUS]: colors.green700,
-    [ROUTES.I_KNOW_A_TEACHER]: colors.tangerine800,
-    [ROUTES.SETTINGS_SECURITY]: colors.ice500,
-};
+const darkTheme = {
+    ...darkThemeWithoutPageBackgroundColors,
```

Comment:
I don't think we should use this pattern of basing the variable names off the theme-independent page background colors or spreading the core theme this way.

Reply:
Same explanation as in the last comment 👍